fdbserver/BlobWorker.actor.cpp

```diff
@@ -2644,12 +2603,16 @@ ACTOR Future<Void> blobWorker(BlobWorkerInterface bwInterf,
 			}
 		}
 	} catch (Error& e) {
+		if (e.code() == error_code_operation_cancelled) {
+			self->granuleMetadata.clear();
```

Comment:
to be safe, explicitly re-throw operation_cancelled after calling clear

Reply:
Ah good idea! throwing here should be fine since it'll be caught by the `Worker`s error forwarder 👍